Automattic/wp-calypso
# Remove: flux likes store

**summary**
We only have 2 remaining usages of the flux likes store.
This PR migrates those two small cases and removes the flux store.


**to test**

the two cases I've migrated here are:

- keyboard navigation likes within a stream. In order to activate you should use j/k to navigate and 
'l' to like
- full post view like/unlike.  to get here, enter a stream. click a card. then try liking/unliking.
- try grepping for `like-store`. nothing should come up
